File: Logica_de_Programacao_Python/Desafio_11/desafio_11/Desafio_11.py
tentativas = 0 

while tentativas < 3:
    usuario = input("Digite o nome usuário: ")
    senha = input("Digite a senha do usuário: ")

    if usuario == "Augusto" and senha == "teste@123":
        print("Bem vindo!")
        break
    else:
        tentativas += 1
        print("Login inválido! Tente novamente.")

    if tentativas == 3:
        print("Usuário bloqueado por tentativas excessivas.")

File: Logica_De_Programacao_Java/Desafio_11/src/Desafio_11.java
import java.util.Scanner;

public class Desafio_11 {
    public static void main(String[] args) {

        Scanner entrada = new Scanner(System.in);
        String usuario, senha;
        int tentativas = 0; // Variável para contar as tentativas

        do {
            System.out.print("Digite o usuário: ");
            usuario = entrada.nextLine();
            System.out.print("Digite a senha: ");
            senha = entrada.nextLine();

            if (usuario.equals("aluno") && senha.equals("segredo")) {
                System.out.println("Bem vindo!");
                break;
            } else {
                tentativas++;
                System.out.println("Login incorreto. Tente novamente.");

                if (tentativas >= 3) {
                    System.out.println("Usuário bloqueado. Número de tentativas excedido.");
                    break;
                }
            }
        } while (true);

        entrada.close();
    }
}